internal/source/kubernetes/source.go

```diff
@@ -29,6 +29,13 @@ import (
 
 var _ source.Source = (*Source)(nil)
 
+var (
+	// configJSONSchema JSON schema with duplications,
+	// as some UI components (e.g. https://github.com/rjsf-team/react-jsonschema-form) don't support nested defaults for definitions.
+	//go:embed config-jsonschema.json
+	configJSONSchema string
```

Comment:
From what I've checked, only the `".*nf_conntrack_buckets.*"` was added. But it was added in a valid place, so what do you mean by:
> JSON schema with duplications

Reply:
I moved this comment from the previous place:

<img width="667" alt="image" src="https://github.com/kubeshop/botkube/assets/7155799/ae0a8366-455d-4bb0-a233-04d8d5e6081e">


Basically, in the JSON schema there are some duplications because the JSON schema form component doesn't support too deeply nested defaults. That's why we couldn't define some nested objects once and refer to them via `$ref`.